core/src/main/java/org/testcontainers/utility/DockerMachineClient.java

```diff
@@ -79,8 +79,8 @@ public void ensureMachineRunning(@NonNull String machineName) {
         }
     }
 
-    public String getDockerDaemonIpAddress(@NonNull String machineName) {
-        return runShellCommand(executableName, "ip", machineName);
+    public String getDockerDaemonUrl(@NonNull String machineName) {
```

Comment:
since this is a public class, please make sure that the old method is available (and make it `@Deprecated`)

Reply:
Done